package.json

```diff
@@ -29,8 +30,8 @@
     "babel-plugin-angularjs-annotate": "^0.10.0",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^3.6.0",
-    "esn-frontend-login": "github:openpaas-suite/esn-frontend-login#main",
     "dotenv-webpack": "^2.0.0",
+    "esn-frontend-login": "github:openpaas-suite/esn-frontend-login#main",
```

Comment:
This does not belong to the commit

Reply:
I used `npm install`, so it auto-rearranged the order of the packages. I think it's fine to keep it this way.